MergerLogic/ImageProcessing/ImageFormatter.cs

```diff
@@ -10,40 +9,24 @@ public enum TileFormat
         [EnumMember(Value = "jpeg")] Jpeg,
     }
 
-    public class ImageFormatter : IImageFormatter
+    public class ImageFormatter
     {
-        public Tile ConvertToFormat(Tile tile, TileFormat format)
+        public static byte[] ConvertToFormat(byte[] tile, TileFormat format)
         {
-            var tileData = tile.GetImageBytes();
-            var currentFormat = this.GetTileFormat(tileData);
-            if (currentFormat != format)
-            {
-                using (var image = new MagickImage(tileData))
-                {
-                    this.ConvertToFormat(image, format);
-                    return new Tile(tile.Z, tile.X, tile.Y, image.ToByteArray());
-                }
-            }
-
-            return tile;
-        }
-
-        public byte[] ConvertToFormat(byte[] tile, TileFormat format)
-        {
-            var currentFormat = this.GetTileFormat(tile);
+            var currentFormat = GetTileFormat(tile);
```

Comment:
with static methods don't we need to call it with the class Name? 
ImageFormatter.GetTileFormat(tile) ?